packages/node-logger/src/generators/node-logger/files/src/index.ts__template__

```diff
@@ -0,0 +1,44 @@
+import winston from 'winston';
```

Comment:
After generating the files into a project, VSCode gives me this error:

![Screenshot 2023-02-06 at 17 37 47](https://user-images.githubusercontent.com/802141/217030343-f751b246-925c-4ed8-9a77-38406eb550ee.png)

Interestingly, we set `esModuleInterop: true` only in `tsconfig.json` inside apps, but not for the whole generated workspace (something we should fix, maybe?).

But in the meantime, maybe it's worth just changing it to

```suggestion
import * as winston from 'winston';
```

to ensure full interoperability?